src/generate_mhq_sample/10_generate_mhq_terr_sample.Rmd

```diff
@@ -1,9 +1,35 @@
 # Introduction
 
+The original data files which are used the scripts are stored in [this google drive folder](https://drive.google.com/open?id=11rStbOup5DFmHP8FQOe_KCOSkXJIwXx0) which can be accessed within INBO.   
 
+# Generation of table with all mhq samples 
```

Comment:
samples or sampling units?

The resulting `samples` data source lists sampling units, not samples, hence may also better be renamed.